.github/workflows/regression-tests.yml

```diff
@@ -11,65 +11,278 @@ on:
 
 jobs:
 
+  lint-whitespace:
+    runs-on: ubuntu-latest
+    name: whitespace
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - name: whitespace
+      run: |
+        ! git --no-pager grep --color --line-number --full-name --extended-regexp -I '\\s+$'
+
+  lint-flake8:
+    runs-on: ubuntu-latest
+    name: flake8
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{matrix.python-version}}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{matrix.python-version}}
+    - name: flake8
+      run: |
+        pip install flake8
+        flake8
+
   tests:
-    name: ${{matrix.sim}} (${{matrix.sim-version}}) | ${{matrix.os}} | Python ${{matrix.python-version}}
+
+    name: ${{matrix.extra_name}}${{matrix.sim}} (${{matrix.sim-version}}) | ${{matrix.os}} | Python ${{matrix.python-version}} ${{matrix.may_fail || '| May Fail'}}
```

Comment:
Does it show `true` when `may_fail` is false?

Reply:
You can see for yourself below. It does not. It shows 'May Fail' at the end or nothing.

Reply:
I think have extra `true`.
"Regression Tests / icarus (master) | ubuntu-20.04 | Python 3.8 **true** (pull_request)" 

It does not matter.

Reply:
You are right, I totally got it backwards. I'll fix it.